doc/Makefile

```diff
@@ -42,8 +49,32 @@ info: $(addprefix ../info/,$(SGMLS:.sgml=.info))
 ../info/%.info: %.sgml | ../info
 	@cd ../info && linuxdoc -B info ../doc/$<
 
+.PRECIOUS: ../docwrk/%.tex
+../docwrk/%.tex: %.sgml | ../docwrk
+	@cd ../docwrk && linuxdoc -B latex ../doc/$<
+
+../docwrk/%.dvi: ../docwrk/%.tex
+	@cd ../docwrk && \
+	rm -f $*.make.log; \
+	(logHash=aaa; \
+	lastLogHash=bbb; \
+	count=1; \
+	echo -n "Generating $*.pdf : "; \
+	while [ "$$lastLogHash" != "$$logHash" ]; \
+	  do \
+	   echo -n "..$$count"; \
+	   count=$$(expr $$count + 1);\
+	   $(LATEX) $< >> $*.make.log; \
+	   lastLogHash=$$logHash; \
+	   logHash=$$(md5sum $*.log); \
+	done;)
+	@echo "..done."
+
+../pdf/%.pdf: ../docwrk/%.dvi | ../pdf
+	@cd ../pdf && dvipdf $<
+
 clean:
-	$(RM) -r ../html ../info
+	$(RMDIR) ../html ../info ../pdf ../docwrk
```

Comment:
The /???wrk directories want to be removed on a mostlyclean too. Please have a look at the other Makefiles.